core/src/main/java/com/oath/oak/OakRValueBufferImpl.java

```diff
@@ -230,4 +219,33 @@ private void lookupValueReference() {
         valueReference = lookUp.valueReference;
         version = lookUp.version;
     }
+
+    /*-------------- OakUnsafeRef --------------*/
+
+    @Override
+    public ByteBuffer getByteBuffer() {
```

Comment:
This is actually the only place I see where we currently rely on good usage of the user. This ByteBuffer should not be accessed while it is written by another thread. Should be noted somewhere...

Reply:
This is now noted in the `OakUnsafeDirectBuffer` interface internal documentation.

Reply:
Should Liran become a Member of the Oak community ? I think he should :)